contrib/guix/guix-clean

```diff
@@ -3,7 +3,7 @@ export LC_ALL=C
 set -e -o pipefail
 
 # Source the common prelude, which:
-#   1. Checks if we're at the top directory of the Bitcoin Core repository
+#   1. Checks if we're at the top directory of the Revolt core repository
```

Comment:
please continue to recheck each of the files under this folder. Thank you.

Reply:
Done changing core to Core